apps/hash-frontend/src/pages/shared/block-collection/component-view.tsx

```diff
@@ -92,7 +92,7 @@ export class ComponentView implements NodeView {
     private readonly block: HashBlock,
     private readonly manager: ProsemirrorManager,
     private readonly readonly: boolean,
-    private readonly isInitialPageTitleEmpty: boolean,
+    private readonly autofocus: boolean,
```

Comment:
Appreciate the PR hasn't introduced this problem, but can we make the casing `autoFocus` everywhere, consistent with the React prop? It's only this casing in a few places in our ProseMirror-related code